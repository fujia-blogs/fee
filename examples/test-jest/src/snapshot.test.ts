import { genConfig } from './snapshot';

test('genConfig', () => {
  expect(genConfig()).toMatchSnapshot({
    time: expect.any(Date),
  });
});
