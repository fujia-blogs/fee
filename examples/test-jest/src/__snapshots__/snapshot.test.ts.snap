// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`genConfig 1`] = `
Object {
  "domain": "fujia.site",
  "port": 6666,
  "time": Any<Date>,
  "url": "https://api.fujia.site",
}
`;
